src/react/components/pages/predict/predictResult.tsx

```diff
@@ -77,6 +94,9 @@ export default class PredictResult extends React.Component<IPredictResultProps,
             </div>
         );
     }
+    private _getMenu(props: IContextualMenuProps): JSX.Element {
```

Comment:
suggestion: change to `getMenu` or `getContextualMenu` for making naming convention consistent.

Reply:
please add a blank line above the private function as well.